apps/stories/src/UseLocalFiles.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import { Meta, Story } from '@storybook/react';
+import { Button, ChakraProvider, Stack, Box, FormLabel } from '@chakra-ui/react';
+import dolbyioTheme from '@millicast-react/dolbyio-theme';
+import * as React from 'react';
+import useLocalFiles from '@millicast-react/use-local-files';
+import VideoView from '@millicast-react/video-view';
+import { useEffect, useState } from 'react';
+
+export default {} as Meta;
+
+export const Default: Story = () => {
```

Comment:
Good to have this story. Could you please also add another story for VideoView using a remote URL?